.zenodo.json

```diff
@@ -3,36 +3,46 @@
         {
             "name": "SuperDARN Data Analysis Working Group"
         },
-        {
-            "affiliation": "Naval Research Laboratory",
-            "name": "Burrell, A.G.",
-            "orcid": "0000-0001-8875-9326"
-        },
         {
             "affiliation": "Dartmouth College",
             "name": "Thomas, E.G.",
             "orcid": "0000-0001-8036-8793"
         },
         {
             "affiliation": "University of Saskatchewan",
-            "name": "Schmidt, M.T.",
-            "orcid": "0000-0002-3265-977X"
+            "name": "Reimer, A.S.",
+            "orcid": "0000-0002-4621-3453"
```

Comment:
@asreimer I listed UoS as the institution since that's where you did the work on `lmfit2`. Please let me know if I should list a different affiliation.